sbt-scalajs-bundler/src/sbt-test/sbt-scalajs-bundler/global-namespace-with-jsdom-unit-testing/common.webpack.config.js

```diff
@@ -2,28 +2,30 @@ var globalModules = {
   moment: "moment"
 };
 
-const importRule = 
-  {
-    // Force require global modules
-    test: /.*-(fast|full)opt\.js$/,
-    loader: "imports-loader?" + Object.keys(globalModules).map(function(modName) {
-      return modName + "=" + globalModules[modName];
-    }).join(',')
-  };
-
+const importRule = {
+  // Force require global modules
+  test: /.*-(fast|full)opt\.js$/,
+  loader:
+    "imports-loader?" +
+    Object.keys(globalModules)
+      .map(function(modName) {
+        return modName + "=" + globalModules[modName];
+      })
+      .join(",")
+};
 
-const exposeRules = 
-  Object.keys(globalModules).map(function(modName) {
-    // Expose global modules
-    return {
-      test: require.resolve(modName),
-      loader: "expose-loader?" + globalModules[modName]
-    }
-  });
+const exposeRules = Object.keys(globalModules).map(function(modName) {
+  // Expose global modules
+  return {
+    test: require.resolve(modName),
+    loader: "expose-loader?" + globalModules[modName]
+  };
+});
 
 const allRules = exposeRules.concat(importRule);
 
 module.exports = {
+  performance: { hints: false },
```

Comment:
It looks like this is the only actual change in that file, right? Other changes shown in the diff are only about code style?

Reply:
This change is to avoid some warnings you get as the size of scala.js bundles are bigger than allowed by default. Other changes are related to styles as my editor uses `prettier` for `.js`. I can remove the style changes if you prefer